view/src/components/surplus.js

```diff
@@ -0,0 +1,556 @@
+          >
+            <AppBar className={classes.appBar}>
+              <Toolbar>
+                <IconButton
+                  edge="start"
+                  color="inherit"
+                  onClick={handleDialogClose}
+                  aria-label="close"
+                >
+                  <CloseIcon />
+                </IconButton>
+                <Typography variant="h6" className={classes.title}>
+                  {this.state.buttonType === "Edit"
+                    ? "Edit Surplus"
+                    : "Create a new Surplus"}
+                </Typography>
+              </Toolbar>
+            </AppBar>
+            <Container maxWidth="lg">
+              <SurplusStepper
+                buttonType={this.state.buttonType}
+                alert={this.props.alert}
+                farmId={this.state.originFarmId}
+                produceId={this.state.produceId}
+                surplusId={this.state.surplusId}
+                handleChange={this.handleChange}
+                handleAsyncChange={this.handleAsyncChange}
+              />
+            </Container>
+          </Dialog>
+          <Container maxWidth="lg">
+            <Grid container spacing={2} alignItem="center">
+              <Grid item xs={12}>
+                <div className={classes.search}>
+                  <div className={classes.searchIcon}>
+                    <SearchIcon />
+                  </div>
+                  <InputBase
+                    fullWidth={true}
+                    placeholder="Search…"
+                    classes={{
+                      root: classes.inputRoot,
+                      input: classes.inputInput,
+                    }}
+                    inputProps={{ "aria-label": "search" }}
+                  />
+                </div>
+              </Grid>
+              {this.state.surplusObjects.map((surplus) => (
+                <Grid item xs={12}>
+                  <Card className={classes.root} variant="outlined">
+                    <CardContent>
+                      <Typography variant="h5" component="h2">
+                        {surplus.totalQuantityAvailable} lbs of
+                        {surplus.produceName} from {surplus.originFarmName}
+                      </Typography>
+                      <Box
+                        display="flex"
+                        flexDirection="row"
+                        flexWrap="wrap"
+                        p={0}
+                        m={0}
+                      >
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Logistics:
+                          </Typography>
+                          <Typography variant="body2" component="p">
+                            Origin: {surplus.originFarmName} (link with card +
+                            info)
```

Comment:
What does "link with card + info" mean here?  Almost looks like a note to self rather than something you intend the user to see.

Reply:
It is a note to self! Tomorrow I'll make sure to take it out